### PR TITLE
Add references to updated System.Net packages

### PR DESCRIPTION
**Pull request by gkhanna79**
@weshaggard @ericstj PTAL

Added the explicit references as discussed offline.

### CODE REVIEW

**Comment by gkhanna79** [comment]
@dotnet-bot test Ubuntu x64 Release Build please

File: pkg/projects/Microsoft.NETCore.App/project.json [hunk 1]
```diff
@@ -39,7 +39,9 @@
     "System.Threading.Tasks.Parallel": "4.0.1",
     "System.Threading.Thread": "4.0.0",
     "System.Threading.ThreadPool": "4.0.10",
-    "Libuv": "1.9.1"
+    "Libuv": "1.9.1",
+    "Microsoft.NETCore.Platforms": "1.0.2-servicing-24703-01",
```
Comment (weshaggard) [inline pkg/projects/Microsoft.NETCore.App/project.json]: nit: I would put these in sort order above but otherwise looks. good.

**Comment by gkhanna79** [comment]
```
09:50:02 Building Docker Container using Dockerfile: scripts/docker/ubuntu.14.04
09:50:02 Cannot connect to the Docker daemon. Is the docker daemon running on this host?
09:50:02 Build step 'Execute shell' marked build as failure
09:50:02 [xUnit] [INFO] - Starting to record.
09:50:02 [xUnit] [INFO] - Processing xUnit.Net-v2 (default)
```

@mmitche Have you seen this before?

CC @dagood @wtgodbe @schellap

File: pkg/projects/Microsoft.NETCore.App/project.json (hunk 1 above)
Comment (gkhanna79) [inline pkg/projects/Microsoft.NETCore.App/project.json]: I kept them out of order for making it explicit that these were getting updated for servicing.

**Comment by gkhanna79** [comment]
@dotnet-bot test Ubuntu x64 Release Build  please

**Comment by gkhanna79** [comment]
Merging since the Ubuntu failure looks like an infra issue.

**Pull request merged**